internal/auth/options.go

```diff
@@ -337,6 +339,8 @@ func AssignStatsdClient(opts *Options) func(*Authenticator) error {
 		switch v := proxy.provider.(type) {
 		case *providers.GoogleProvider:
 			v.SetStatsdClient(StatsdClient)
```

Comment:
We should probably just add this to the interface at this point so we don't have to do the type switching here.